src/AppBundle/Resources/javascript/AuthenticationPageService.ts

```diff
@@ -90,6 +90,7 @@ export class AuthenticationPageService {
     this.statusErrorComponent.show();
   }
 
+  /* istanbul ignore next */
```

Comment:
TIL: `istanbul ignore next` instructs code coverage reporting to ignore this block.

Is that really necessary, as you removed the `coverageThreshold` from the jest config?